components/Milestone.vue

```diff
@@ -95,16 +98,15 @@ export default {
   },
   methods: {
     updateDeadline() {
-      this.dateRelative = formatRelative(this.milestone.date, new Date(), {
+      this.deadlineRelative = formatRelative(this.milestone.date, new Date(), {
```

Comment:
This should have been `deadlineRelative`, otherwise the milestones don't display their deadline